src/provider/client.ts

```diff
@@ -44,24 +41,12 @@ export class APIClient {
     }
   }
 
-  public async verifyAuthentication(): Promise<void> {
-    try {
-      const { id } = await this.getData('/current_customer');
-      if (id !== this._config.customerId) {
-        throw new IntegrationProviderAuthenticationError({
-          endpoint: '/current_customer',
-          status: 200,
-          statusText: 'Incorrect Customer Id',
-        });
-      }
-    } catch (err) {
-      throw new IntegrationProviderAuthenticationError({
-        cause: err,
-        endpoint: err.url,
-        status: err.status,
-        statusText: err.statusText,
-      });
-    }
+  public async getCurrentCustomer(): Promise<FastlyAccount> {
```

Comment:
A few questions:
1. Looks like you removed the error that was thrown previously ('Incorrect Customer Id'). Is that of any concern?
2. You defined `getCurrentCustomer()` in its place, but it is unused in the code. I'd get rid of it if it's unnecessary.

Reply:
1. I don't think this check was ever necessary. The customer ID is currently supplied as part of the integration config, but it can be derived from the `current_user` API. We should change the integration config to only require an API key and remove customer ID.
2. Will remove